### ISSUE TITLE
onCancel event doesn't fire

### ISSUE THREAD

**Issue by prettyv** [running-coder/jquery-typeahead#289]
Since @f580622a40663ddbd1b8089c7fe04684a73eae61, the onCancel event doesn't fire anymore, because
```
resetInput: function () {
    this.node.val('');
    this.item = null;
    this.query = '';
    this.rawQuery = '';
},
```
hinders
```
// #195 Trigger an onCancel event if the Typeahead is cleared
if (scope.rawQuery === "" && scope.query !== "") {
    originalEvent = originalEvent || {};
    scope.helper.executeCallback.call(scope, scope.options.callback.onCancel, [scope.node, e]);
}
```
from working and is executed before the 'input' event is triggered.

**Comment by running-coder** [comment]
hey thanks for flagging the issue, it was fixed in #274 waiting for the 2.8.0 release, if you need that fix you can use the dev branch for now

**Comment by prettyv** [comment]
Thanks for the quick response! Apparently I overlooked the previously closed issue - using the current version from the dev-branch seems to work just fine for now indeed.
